src/featureFlags/index.tsx

```diff
@@ -25,12 +25,12 @@ export enum FeatureFlag {
 
 interface FeatureFlagsContextType {
   isLoaded: boolean
-  flags: Record<string, string>
+  flags: Record<string, any> // includes flags & configs
```

Comment:
why `any`? can a config be typed as `Record<string, string>` or something similar?
```suggestion
  flags: Record<string, string | Record<string, string>> // includes flags & configs
```

Reply:
hmm config is a json object so hard to say what it would be used for -- in this case we're storing something like `{ chains: [1, 3, 10] }` which is Record<string, ChainId[]>